convert2rhel/checks.py

```diff
@@ -45,11 +51,14 @@
 
 def perform_pre_checks():
     """Early checks after system facts should be added here."""
+
     check_efi()
     check_tainted_kmods()
     check_readonly_mounts()
     check_rhel_compatible_kernel_is_used()
     check_custom_repos_are_valid()
+    check_package_updates()
+    is_loaded_kernel_latest()
```

Comment:
This would be the correct order for these checks?

Reply:
Yes. But please move the following from main.py to run before the `perform_pre_checks()` call:
```
loggerinst.task("Prepare: Clear YUM/DNF version locks")
pkghandler.clear_versionlock()
```
The reason is that when [yum versionlock](https://access.redhat.com/solutions/98873) is in place for kernel, `yum update` will not report the kernel to be updated even when a newer version is available in repos.